test/thrift/generator/service_test.exs

```diff
@@ -370,8 +370,6 @@ defmodule Thrift.Generator.ServiceTest do
 
   thrift_test "clients exit if they try to use a closed client", ctx do
     Process.flag(:trap_exit, true)
-
-    ref = Process.monitor(ctx.server)
```

Comment:
@scohen you are correct: https://github.com/pinterest/elixir-thrift/pull/263#issuecomment-292737382